pkg/reconciler/broker/broker_test.go

```diff
@@ -50,6 +52,7 @@ import (
 	v1a1addr "knative.dev/pkg/client/injection/ducks/duck/v1alpha1/addressable"
 	v1b1addr "knative.dev/pkg/client/injection/ducks/duck/v1beta1/addressable"
 	fakekubeclient "knative.dev/pkg/client/injection/kube/client/fake"
+	"knative.dev/pkg/client/injection/kube/informers/core/v1/service"
```

Comment:
This could be aliased to `serviceinformers` for consistency (and because `service.Key{}` feels kind of off to me 🙂)